tests/tasks/setup/eks/awscli-cfn-lt.yaml

```diff
@@ -0,0 +1,69 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: awscli-eks-cfn-launch-template-create
```

Comment:
could you trim `create` from the suffix ?

Reply:
Removed this